### ISSUE TITLE
(2) Apply parent-child metadata inheritance rules to collection-works

### ISSUE THREAD

**Issue by rmjaffe** [UCSCLibrary/dams_project_mgmt#308]
### Descriptive Summary

As a metadata librarian, I would like to see collection-level metadata values be inherited by works within that collection.  These inheritance rules would be the same at the rules in place for parent and child works.

### Background 

While reviewing the Ansel Adams collection within the DAMS, I noticed that the images within the collection did not appear to have inherited collection-level metadata values.  Specifically, collection-level values where not inherited where values were absent in the work metadata (e.g. any of the rights properties or subjectPlace) or collection-level subject values were not added to the values already appearing in the work metadata (e.g. subjectName). 

Metadata inheritance is working at expected for parent work-child works, but I don't think we had previously specifically called out applying these inheritance rules for collection-work relationships as part of the work on implementing collection-level metadata.

Inheritance rules will need to be better documented as part of work in the configuration file -- likely a separate ticket will need to be created for this #77 Config file overhaul.

### Acceptance Criteria

This is what done looks like:

* [ ] Works are inheriting collection-level values

### Related Work 

#154 Customize collection level metadata schema to match work level schema
This work will need to be complete before #309 Re-ingest Ansel Adams
